dashboard/app/serializers/user_serializer.rb

```diff
@@ -72,7 +72,8 @@
 #
 
 class UserSerializer < ActiveModel::Serializer
-  attributes :id, :email, :ops_first_name, :ops_last_name, :district, :ops_school, :ops_gender
+  attributes :id, :email, :ops_first_name, :ops_last_name, :district, :ops_school, :ops_gender,
+             :races, :race_interstitial_shown
```

Comment:
It'll break staging if submitted as is, so it should be changed. Hound wants you to indent `:races` two spaces relative to the previous line.